ptf_tests/tests/LNT_ECMP_2HotPlugVM_2Host_netperf.py

```diff
@@ -0,0 +1,523 @@
+# Copyright (c) 2022 Intel Corporation.
+#
```

Comment:
Add SPDX license info line 

`# SPDX-License-Identifier: Apache-2.0`

For future reference, please add the SPDX line for all new files added.